src/embedded/index.ts

```diff
@@ -67,11 +67,15 @@ export class EmbeddedOptions {
     if (!cfg || !cfg.version) {
       return defaultVersion;
     }
+    if (cfg.version == 'latest') {
+      return 'latest';
+    }
     if (cfg.version.match(/[1-9]\.[1-9]{2}\..*/g)) {
       return cfg.version;
-    } else {
-      throw new Error(`invalid version: ${cfg.version}. version must resemble '{major}.{minor}.{patch}'`);
     }
+    throw new Error(
+      `invalid version: ${cfg.version}. version must resemble '{major}.{minor}.{patch}, or 'latest'`
+    );
```

Comment:
I think we still should support URLs. In the future we might build binaries for each PR

Reply:
done